packages/nuxeo-web-ui-ftest/features/step_definitions/support/fixtures/layouts.js

```diff
@@ -171,15 +171,28 @@ global.fieldRegistry.register(
 );
 global.fieldRegistry.register(
   'nuxeo-checkbox-aggregation',
-  (element) => element.element('paper-checkbox').getAttribute('aria-checked'),
+  (element) => {
+    let el = element;
+    if (el.getAttribute('collapsible')) {
+      el = el.element('iron-collapse');
+    }
+    el.element('paper-checkbox').getAttribute('aria-checked');
+  },
   (element, value) => {
-    element.waitForVisible('paper-checkbox');
-    const els = element.elements('paper-checkbox').value;
-    const el = els.find((e) => {
+    let el = element;
+    if (el.getAttribute('collapsible')) {
+      el = el.element('iron-collapse');
+      const button = element.element('button');
+      button.waitForVisible();
+      button.click();
+      el.element('iron-collapse').waitForVisible();
+    }
+    const els = el.elements('paper-checkbox').value;
```

Comment:
We should wait for the paper-checkbox to be visible first:
```suggestion
    el.waitForVisible('paper-checkbox');
    const els = el.elements('paper-checkbox').value;
```